packages/builder/src/components/backend/DataTable/DataTable.svelte

```diff
@@ -27,6 +27,17 @@
   $: isUsersTable = id === TableNames.USERS
   $: isInternal = $tables.selected?.type !== "external"
 
+  $: datasource = $datasources.list.find(datasource => {
+    return datasource._id === $tables.selected.sourceId
+  })
+
+  const relationshipSupport = datasource => {
+    const integration = $integrations[datasource?.source]
+    return !isInternal && integration?.relationships !== false
+  }
+
+  $: relationshipsEnabled = relationshipSupport(datasource)
```

Comment:
NAB: I tend to group reactive statements together for readability, and automatic code hosting will take care of the order (as in it's ok to have the reactive statement before the function definition - it'll still be able to use it).